reference/config_files/conandata.yml.rst

```diff
@@ -0,0 +1,50 @@
+.. _reference_config_files_conandata_yml:
+
+conandata.yml
```

Comment:
Uhm, I'd probably do not put this as config-file. I'd move this section to ``conanfile.py`` section.